tests/Testing/TestResponseTest.php

```diff
@@ -488,6 +488,36 @@ public function testAssertUnprocessable()
         $response->assertUnprocessable();
     }
 
+    public function testAssertBadRequestWhenNotBadRequest()
+    {
+        $statusCode = 500;
+
+        $this->expectException(AssertionFailedError::class);
+
+        $this->expectExceptionMessage('Expected response status code');
+
+        $baseResponse = tap(new Response, function ($response) use ($statusCode) {
+            $response->setStatusCode($statusCode);
+        });
+
+        $response = TestResponse::fromBaseResponse($baseResponse);
+        $response->assertBadRequest();
```

Comment:
I noticed that this set of assertions was repeated a lot in this test class.  It was outside the scope of this PR, but would people be interested in maybe converting some of these to using a `dataProvider`?  Not 100% sure how it would look yet but it may help DRY this up.

Reply:
Obviously, that would be a separate PR